index.html

```diff
@@ -940,19 +945,37 @@
 
 ### Rights {#data-rights}
 
-To the extent possible, [=people=] should retain a certain number of rights over their data, and
-these rights should be facilitated by the [=parties=]. While data rights are a self-governing
-mechanism that is not sufficient to support [=privacy=], they do complement self-determination and
-help improve accountability. Such rights should include:
+[=People=] retain certain rights over data about themselves, and these rights should be facilitated by their [=user agent=] and the [=parties=]. While data rights alone are not sufficient to support [=privacy=], they do support self-determination and help improve accountability. Such rights include:
 
 * The right to <em>access</em> [=data=] about oneself.
-* The right to <em>erase</em> [=data=] about oneself, with or without terminating the service (though
-  what [=data=] can be erased may differ between those two cases).
-* The right to <em>port</em> information that one has stored with a given [=party=] in such a way
-  that it can easily be reused or transferred elsewhere.
+
+  This right includes being able to discover data about oneself (no secret databases) and to review what information has been collected or inferred.
```

Comment:
```suggestion
  This right includes being able to discover data about oneself (implying no databases are kept secret) and to review what information has been collected or inferred.
```

Reply:
I was going for the pithy tagline, but this does seem clearer, yes.